### ISSUE TITLE
fine-tuning parameters

### ISSUE THREAD

**Issue by tzuyi0414** [halsay/MFCC_tutorial#1]
I have WAV files in two languages, one in Chinese and the other in English. I'd like to ask if I need to adjust parameters (like frame length) when processing these files?

**Comment by halsay** [comment]
No, I dont think you need to change the parameters. Just use MFCC as a compact feature for your task (e.g. ASR, TTS, LID).

**Comment by tzuyi0414** [comment]
Thank you very much. It helps me a lot!!
